docs/analyzer/recognizer_registry_provider.md

```diff
@@ -71,19 +71,23 @@ The recognizer list comprises of both the predefined and custom recognizers, for
         - language: en
         context: [credit]
         - language: es
-        context: [tarjeta,credito]
+        context: [zip, code]
     supported_entity: "ZIP"
-    deny_list: [Mr., Mrs., Ms., Miss, Dr., Prof.]
-    deny_list_score: 1
     type: custom
     enabled: true
+
+    - name: "TitlesRecognizer"
+    supported_language: "en"
+    supported_entity: "TITLE"
+    deny_list: [Mr., Mrs., Ms., Miss, Dr., Prof.]
+    deny_list_score: 1
 ```
 
 The recognizer parameters:
 
   - `supported_languages`: A list of supported languages that the analyzer will support. In case this field is missing, a recognizer will be created for each supported language provided to the `AnalyzerEngine`. 
   In addition to the language code, this field also contains a list of context words, which increases confidence in the detection in case it is found in the surroundings of a detected entity (as seen in the credit card example above).
-  - `type`: this could be either predefined or custom.
+  - `type`: this could be either predefined or custom. As this is optional, if not stated otherwise, the default type is custom.
```

Comment:
Is it custom? If I write a name of a recognizer (say, CreditCardRecognizer), wouldn't it load the predefined one?

Reply:
You're right! It goes like this: 
1. A single string is predefined because it'll try and load the class instance with defaults
2. once you add up parameters the recognizer is custom by default. The reason for it is backward compatibility - previously everything was custome, and we don't wanna break the functionality.

Reply:
Actually, this is a little confusing. Let's think of a way to elaborate the difference. I already see users getting confused around this flag.